addons/website_sale_stock/models/res_config_settings.py

```diff
@@ -1,43 +1,36 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models, api
+from odoo import fields, models
+from odoo.tools.translate import html_translate
 
 
 class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
-    allow_out_of_stock_order = fields.Boolean(
-        string='Continue selling when out-of-stock',
-        default=True)
-    available_threshold = fields.Float(
-        string='Show Threshold',
-        default=5.0)
-    show_availability = fields.Boolean(
-        string='Show availability Qty',
-        default=False)
+    default_allow_out_of_stock_order = fields.Boolean(
+        string="Continue selling when out-of-stock",
+        default=True,
+        default_model='product.template',
+    )
+    default_out_of_stock_message = fields.Html(
+        string="Out-of-Stock Message",
+        translate=html_translate,
```

Comment:
Translation doesn't seem to work, and I doubt creating `ir.default` records with a json value for multiple languages even works.

@Jean-boje is this new default meaningful in the settings if we can only set the default value for the main language ?